docs/CoC.md

```diff
@@ -0,0 +1,211 @@
+essential to counteract the many forms of inequality and abuses of power that
+exist in societies everywhere.
+
+As such, Pext is committed to the goal of open culture. Specifically, our
+community ensures that:
+
+* All source code is available under an open-source license.
+* All documentation, including artwork and logos, are available under an open
+  culture license.
+* Copyleft licenses are employed throughout to guarantee independence from
+  corporate influence and to preserve the benefits of any modifications for all.
+* Everything created by our community is available in an accessible way, without
+  DRM or other artificial restrictions to use and participation.
+
+We require all participants in our community to consider and abide by the
+principles above, in the spirit in which they are included in this Code of
+Conduct.
+
+## Expected behaviour
+
+The following behaviours are expected and requested of all community members:
+
+* Participate in an authentic and active way. In doing so, you contribute to the
+  health and longevity of this community.
+* Exercise consideration and respect in your speech and actions.
+* Attempt collaboration before conflict.
+* Refrain from demeaning, discriminatory and harassing behaviour and speech.
+* Be mindful of your surroundings and your fellow participants. Alert community
+  leaders if you notice a dangerous situation, someone in distress, or
+  violations of this Safety and Well-Being Policy, even if they seem
+  inconsequential.
+* Remember that community venues, both physical and digital, may be shared with
+  members of the public. Please be respectful to all patrons of such locations.
+
+## Unacceptable behaviour
+
+The following behaviours are considered harassment and are unacceptable within
+our community:
+
+* Violence, threats of violence or violent language directed against another
+  person. Note that misgendering, especially intentional misgendering, is a
+  form of violent language and will not be tolerated.
+* Sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory
+  jokes and language.
+* Posting or displaying sexually explicit or violent material.
+* Posting or threatening to post other people's personally identifying
+  information ("doxing").
+* Personal insults, particularly those related to gender, sexual or romantic
+  orientation, ethnicity, religion or disability.
+* Inappropriate photography or recording. Note that photography or recording
+  without prior consent is _always_ inappropriate.
+* Inappropriate physical contact. You must have someone's consent before
+  touching them.
+* Unwelcome sexual attention. This includes, but is not limited to: sexualized
+  comments or jokes, inappropriate touching, groping and unwelcome sexual
+  advances. If you do not have someone's prior consent, then any kind of sexual
+  attention is unwelcome.
+* Deliberate intimidation, stalking or following (online or in person).
```

Comment:
Hmm, I get what you mean here, but "following" sounds confusing as there is a follow button on GitHub. Maybe just simplify it to "Deliberate intimidation or stalking (online or in person)."?

Reply:
Yeah, that is a little bit vague and probably unnecessary. Will remove the 'following' part.